client/lib/analytics/index.js

```diff
@@ -577,9 +569,7 @@ const analytics = {
 			if ( window.location ) {
 				const parsedUrl = url.parse( window.location.href );
 				const urlParams = parse( parsedUrl.query );
-				const utmParams = pickBy( urlParams, function( value, key ) {
-					return startsWith( key, 'utm_' );
-				} );
+				const utmParams = pickBy( urlParams, ( value, key = '' ) => key.startsWith( 'utm_' ) );
```

Comment:
Can the `key` ever be anything else than a valid string?

Reply:
I'm not sure what could come out of the previous expression. Since I was replacing `_.startsWith` with the native method on strings with the same name, I figured I'd ensure it really is a string by using a default value.

Reply:
`pickBy` works on an object (and converts the collection to object if it isn't one yet) and object keys are always strings or symbols. The `''` default wouldn't save you from symbol here, and a numeric key (if it was possible) would crash on `key.startsWith`, too.

I think just `( value, key ) => key.startsWith( 'utm_' )` covers everything that can realistically happen here 🙂

Reply:
Perfect, the simpler the better 🙂 Thanks!